solidity/contracts/system/TBTCSystem.sol

```diff
@@ -38,9 +38,9 @@ contract TBTCSystem is Ownable, ITBTCSystem, DepositLog {
         uint16 _severelyUndercollateralizedThresholdPercent,
         uint256 _timestamp
     );
-    event KeepFactorySingleShotUpdateStarted(
+    event KeepFactoryUpdateStarted(
```

Comment:
This should become `KeepFactories...`, I think.

Reply:
I'm glad you mentioned that, I was thinking about the same.

Reply:
99fca0836140503a797bd509d913adf86dbdd33b